xbmc/addons/AddonManager.h

```diff
@@ -185,6 +189,15 @@ namespace ADDON
      */
     bool IsAddonDisabled(const std::string& ID) const;
 
+    /* \brief Check whether an addon has been disabled via DisableAddon except for a particular reason
+     In case the disabled cache does not know about the current state the database routine will be used.
+     \param ID id of the addon
+     \param disabledReason the reason that will be an exception to being disabled
+     \sa DisableAddon
+     */
```

Comment:
Also here with `*` on each line, also after `@brief` an new line if text below not related to the first line. (I also prever to use `@` and the `[in]` or `[out]` part :smile:)